includes/discovery/sensors/dbm/mrv-optidriver.inc.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * LibreNMS Interface Power dBm module for the MRV® OptiDriver® Optical Transport Platform
+ *
+ * © 2017 Chris A. Evans <thecityofguanyu@outlook.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+echo 'MRV OptiDriver ';
+
+$multiplier = 1;
+$divisor = 1000;
+$limit_low = -100;
```

Comment:
Multiplier and divisor are right, but those limits are definitely not sane. I'll see if I can find some documentation to set those accordingly.